jujugui/static/gui/src/app/app.js

```diff
@@ -1959,11 +1959,12 @@ YUI.add('juju-gui', function(Y) {
       @param {Function} next - Call to continue dispatching.
     */
     _renderStatusView: function(state, next) {
+      const propTypes = window.juju.components.Status.propTypes;
```

Comment:
Is two times enough? I am not sure, and it would break inspect-component. We need to update that script to support dynamic rebinding as this one you are suggesting.